src/pages/workspaces/workspace/Data.js

```diff
@@ -762,6 +765,13 @@ const WorkspaceData = _.flow(
                 wdsSchemaError && h(NoDataPlaceholder, {
                   message: 'WDS is unavailable.'
                 }),
+                !wdsSchemaError && !_.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) && _.isEmpty(wdsSchema) && h(NoDataPlaceholder, {
+                  message: 'No WDS tables have been uploaded.'
+                }),
```

Comment:
I don't think the `!_.some({ targetWorkspace: { namespace, name } }, asyncImportJobs)` clause is relevant here. We're not using Import Service for WDS, so this is irrelevant.